meeting-notes/2019-Jun-06.md

```diff
@@ -0,0 +1,47 @@
+# Falco Office hours meeting 6th June 2019
+
+**MC:** Lorenzo Fontana
+
+**Who joined**:
+
+- Lorenzo Fontana ([@fntlnz](https://github.com/fntlnz))
+- Mark Stemm
+- Michael Ducy
+- Leonardo Di Donato
+- Spencer Krum
+- Thomas Labarussias ([@issif](https://github.com/issif)]
+- Mattia Lavacca
+- Radhika Puthiyetath
+
+This was the first official Office Hours meeting! We had two proposals to discuss
+and there have been 7 people participating very actively.
+
+## Proposals
+
+[#5](https://github.com/falcosecurity/office-hours/issues/5): Improved Falco outputs
+
+Thomas Labarussias presented it as proposed in the issue, slightly different than what was proposed initially on having a grpc output.
+
+- **Ducy:** thinks that it should be only a  grpc plugin and not a sidekick 
+- **Leo:** Also if I’d like to have fan-in/out mechanisms and broadcasting of events, I think the architecture is a bit overkill, for example I have concerns about all the queues in the diagram and the different mechanisms used to grab data from them (push vs pull)
+- Mark and Kaizhe think that the architecture is a bit giant, needs to be simpler, the diagram
+- **Kaizhe Question:** when I look into the diagram I can’t understand what Falco is expected to provide, autiding events or what? It’s not that crazy now, trying to reduce noise, what do you expect from Falco regarding events.
+- **Thomas Answer:** I just want a very simple configuration on which plugins are enabled and what events are received, if you take my program I didn’t integrate the idea of minimum criticity, it’s only that, which plugin are enabled and events. No rules on which kind of messages etc.. Falco has to be the truth, the base of knowledge.
+- **Kaizhe:** I’m more looking into this to enrich how this new architecutre can bring the detection capabilities to the next Level, I see some flexibility here when we have a broker and it can’t be easily enrich the Falco events but I think I still agree with Mark, maybe start with simple and see what we can do from there
+- **Thomas:** It’s a little bit complicated to have a mechanism that will work everywhere
+** Mattia:** I don’t have a complete understanding and view on this proble, just started to learn falco last week.
+
+[#6](https://github.com/falcosecurity/office-hours/issues/6): Github and contribution automation:
+
+- **Leonardo:** we have an idea to improve the way internal and external contribute to falco, this idea is to implement and automate a lot the workflow, to make everything more structured in a way that in every moment we have statistics about the number of issues, the kind of issues, the area the issue refers to, and a set of commands, and a set of well defined rules so that for example no one can just merge their PR without following the rules. 
```

Comment:
```suggestion
- **Leonardo**: we have an idea to improve the way internal and external contribute to falco, this idea is to implement and automate a lot the workflow, to make everything more structured in a way that in every moment we have statistics about the number of issues, the kind of issues, the area the issue refers to, and a set of commands, and a set of well defined rules so that for example no one can just merge their PR without following the rules. 
```